### PR TITLE
Update README.md for OS X users.

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java

### CODE REVIEW

**Comment by stk1m1** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -87,6 +87,12 @@ Prerequisites:
    export REALM_CORE_DOWNLOAD_DIR=~/.realmCore
    ```
 
+   OS X users should also add following line to `~/.profile` in order for Android Studio to see this environment variable..
```
Comment (cmelchior) [inline README.md]: I think this is only true if you use the default terminal right? I think @emanuelez and @beeender have been complaining about it not working.

Comment (zaki50) [inline README.md]: right. `.zprofile` instead of `.profile` must be used for `zsh`.

Comment (zaki50) [inline README.md]: @cmelchior updated

**Comment by dalinaum** [comment]
:+1:

**Comment by emanuelez** [comment]
:+1:

**Pull request merged**